client/src/app/account/account-videos/account-videos.component.ts

```diff
@@ -26,6 +26,10 @@ export class AccountVideosComponent extends AbstractVideoList implements OnInit,
     itemsPerPage: 5,
     totalItems: null
   }
+  syndication = {
+    isSupported: false,
```

Comment:
I don't think you need this `isSupported`, just do ngIf on `syndicationItems.length`